README.md

```diff
@@ -7,8 +7,20 @@ The Yellow Paper is a formal definition of the Ethereum protocol, originally by
 
 It is free culture work, licensed under the Creative Commons Attribution Share-Alike (CC-BY-SA) version 4.0.
 
-## Usage
+## Releases
+This paper currently does not include changes to Ethereum after Byzantium, however if you like, feel free to make a PR to the [`byzantium` branch](https://github.com/ethereum/yellowpaper/tree/byzantium)! For further information on past hard forks, see [here](https://github.com/ethereum/EIPs/#past-hard-forks) as well as [here](https://github.com/ethereum/wiki/wiki/Releases) for previous and planned releases.
```

Comment:
This is now outdated. As for a reference to various forks, this PR also tries to cover that https://github.com/ethereum/yellowpaper/pull/406